packages/core/src/fetchers/sideloading.spec.ts

```diff
@@ -59,15 +59,7 @@ describe('sideloadPrebindingDefaultValues', () => {
                       },
                     },
                   },
-                  contentTypes: {
-                    ct111: {
-                      sys: {
-                        id: 'ct111',
-                        type: 'Link',
-                        linkType: 'ContentType',
-                      },
-                    },
-                  },
+                  contentTypes: ['ct111'],
```

Comment:
Not sure if this was already discussed before but did you consider `contentTypeIds`? `contentTypes` indicates a list of content types, to me personally at least.